lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -94,13 +98,39 @@ export default function FilePickerApp({ onSubmit }) {
   // Submit the form after a selection is made via one of the available
   // methods.
   useEffect(() => {
+    async function createAssignment() {
```

Comment:
Should this be defined outside as a const?

Reply:
You can do, although you'd need to use [useCallback](https://reactjs.org/docs/hooks-reference.html#usecallback) to help Preact keep track of the effect's dependencies. This is fine for the moment.